crc32.c

```diff
@@ -268,12 +275,187 @@ local unsigned long crc32_generic(crc, buf, len)
     return crc ^ 0xffffffffUL;
 }
 
+
 #ifdef HAS_PCLMUL
 
+//https://github.com/webosose/chromium68/blob/master/src/third_party/zlib/crc32_simd.c
```

Comment:
Maybe instead of pointing to an outdated copy of Chromium (M68), you could reference the ToT (Top of Tree) original repository Chromium repo*? 

Since M68, I've added some new optimizations, refactorings and bug fixes on Chromium's zlib.
:-)

*https://cs.chromium.org/chromium/src/third_party/zlib/